.github/workflows/nightly.yml

```diff
@@ -8,6 +8,10 @@ on:
   schedule:
   - cron: '0 3 * * *'
 
+env:
+  CLAZY_CHECKS: "level2,no-qstring-allocations,no-fully-qualified-moc-types"
```

Comment:
https://invent.kde.org/sdk/clazy/-/merge_requests/111 just landed so we don't need to exclude fully-qualified-moc-types anymore -- assuming you can somehow make CI use the latest clazy from git master.

Reply:
I use distro packages, if there were prebuilt clazy would likely be easy, we can also compile it here if you think it's worth it?

Reply:
Given the low amount of development on KDSoap, it's fine to exclude it until the distro has a fixed clazy.